technic/machines/LV/music_player.lua

```diff
@@ -93,9 +94,11 @@ minetest.register_node("technic:music_player", {
 	tiles = {"technic_music_player_top.png", "technic_machine_bottom.png", "technic_music_player_side.png",
 	         "technic_music_player_side.png", "technic_music_player_side.png", "technic_music_player_side.png"},
 	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,
-		technic_machine=1, technic_lv=1},
+		  technic_machine=1, technic_lv=1, axey = 2, handy=1},
```

Comment:
Minor indent / style
```suggestion
		technic_machine=1, technic_lv=1, axey = 2, handy=1},
```